GCC 15 maybe uninitialised warning

With CUDA 13.0 and GCC 15 a new `-Wmaybe-uninitialized` warning is encountered while building the test suite.

```
In function ‘void* memcpy(void*, const void*, size_t)’,
    inlined from ‘std::array<T, N> flamegpu::EnvironmentDescription::setProperty(const std::string&, const std::array<T, N>&) [with T = unsigned char; unsigned int N = 10]’ at /home/ptheywood/code/flamegpu/FLAMEGPU2/include/flamegpu/model/EnvironmentDescription.h:529:7,
    inlined from ‘void flamegpu::_GLOBAL__N__9f03b52a_31_test_environment_description_cu_291c9867_11998::ExceptionPropertyLength_test() [with T = unsigned char]’ at /home/ptheywood/code/flamegpu/FLAMEGPU2/tests/test_cases/model/test_environment_description.cu:197:10:
/usr/include/x86_64-linux-gnu/bits/string_fortified.h:29:30: warning: ‘_b3’ may be used uninitialized [-Wmaybe-uninitialized]
   29 |   return __builtin___memcpy_chk (__dest, __src, __len,
      |        ~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~                                       
/home/ptheywood/code/flamegpu/FLAMEGPU2/tests/test_cases/model/test_environment_description.cu: In function ‘void flamegpu::_GLOBAL__N__9f03b52a_31_test_environment_description_cu_291c9867_11998::ExceptionPropertyLength_test() [with T = unsigned char]’:
/home/ptheywood/code/flamegpu/FLAMEGPU2/tests/test_cases/model/test_environment_description.cu:190:37: note: ‘_b3’ declared here
  190 |     std::array<T, ARRAY_TEST_LEN * 2> _b3;
      |                                     ^~~
In function ‘void* memcpy(void*, const void*, size_t)’,
    inlined from ‘std::array<T, N> flamegpu::EnvironmentDescription::setProperty(const std::string&, const std::array<T, N>&) [with T = signed char; unsigned int N = 10]’ at /home/ptheywood/code/flamegpu/FLAMEGPU2/include/flamegpu/model/EnvironmentDescription.h:529:7,
    inlined from ‘void flamegpu::_GLOBAL__N__9f03b52a_31_test_environment_description_cu_291c9867_11998::ExceptionPropertyLength_test() [with T = signed char]’ at /home/ptheywood/code/flamegpu/FLAMEGPU2/tests/test_cases/model/test_environment_description.cu:197:10:
/usr/include/x86_64-linux-gnu/bits/string_fortified.h:29:30: warning: ‘_b3’ may be used uninitialized [-Wmaybe-uninitialized]
   29 |   return __builtin___memcpy_chk (__dest, __src, __len,
      |        ~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~                                       
/home/ptheywood/code/flamegpu/FLAMEGPU2/tests/test_cases/model/test_environment_description.cu: In function ‘void flamegpu::_GLOBAL__N__9f03b52a_31_test_environment_description_cu_291c9867_11998::ExceptionPropertyLength_test() [with T = signed char]’:
/home/ptheywood/code/flamegpu/FLAMEGPU2/tests/test_cases/model/test_environment_description.cu:190:37: note: ‘_b3’ declared here
  190 |     std::array<T, ARRAY_TEST_LEN * 2> _b3;
      |                                     ^~~
```

--- a/tests/test_cases/model/test_environment_description.cu
+++ b/tests/test_cases/model/test_environment_description.cu
@@ -186,8 +186,8 @@ void ExceptionPropertyLength_test() {
     EnvironmentDescription ed = model.Environment();
     std::array<T, ARRAY_TEST_LEN> b;
     std::array<T, 1> _b1 = {};
-    std::array<T, ARRAY_TEST_LEN + 1> _b2;
-    std::array<T, ARRAY_TEST_LEN * 2> _b3;
+    std::array<T, ARRAY_TEST_LEN + 1> _b2 = {};
+    std::array<T, ARRAY_TEST_LEN * 2> _b3 = {};
     ed.newProperty<T, ARRAY_TEST_LEN>("a", b);
     auto fn1 = &EnvironmentDescription::setProperty<T, 1>;
     auto fn2 = &EnvironmentDescription::setProperty<T, ARRAY_TEST_LEN + 1>;

